plugin/adsk/plugin/importTranslator.cpp

```diff
@@ -58,7 +58,6 @@ UsdMayaImportTranslator::reader(
         MPxFileTranslator::FileAccessMode  /*mode*/)
 {
     std::string fileName(file.fullName().asChar(), file.fullName().length());
-    std::map<std::string, std::string> variants;
```

Comment:
Hmm.... I am wondering why this was not caught by the compiler?

Reply:
indeed!